### PR TITLE
Add openvino and neural-compressor to setup

### PR DESCRIPTION
**Pull request by echarlaix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ As such, Optimum enables users to efficiently use any of these platforms with th
...
 - [Graphcore IPUs](https://github.com/huggingface/optimum-graphcore) - IPUs are a completely new kind of massively parallel processor to accelerate machine intelligence. More information [here](https://www.graphcore.ai/products/ipu).
 - [Habana Gaudi Processor (HPU)](https://github.com/huggingface/optimum-habana) - [HPUs](https://docs.habana.ai/en/latest/Gaudi_Overview/Gaudi_Architecture.html) are designed to maximize training throughput and efficiency. More information [here](https://habana.ai/training/).
-- [Intel](https://github.com/huggingface/optimum-intel) - Enabling the usage of Intel tools to accelerate end-to-end pipelines on Intel architectures. More information [here](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html).
+- [Intel](https://github.com/huggingface/optimum-intel) - Enabling the usage of Intel tools to accelerate end-to-end pipelines on Intel architectures. More information [here](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html) and [here](https://docs.openvino.ai/latest/index.html).
```
Comment (JingyaHuang) [inline README.md:18]:
```suggestion
- [Intel](https://github.com/huggingface/optimum-intel) - Enabling the usage of Intel tools to accelerate end-to-end pipelines on Intel architectures. More information about [Intel Neural Compressor](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html) and [OpenVINO](https://docs.openvino.ai/latest/index.html).
```

File: README.md [hunk 2]
```diff
@@ -43,12 +43,13 @@ python -m pip install optimum
...
-| [Graphcore IPU](https://www.graphcore.ai/products/ipu)                                                                 | `python -m pip install optimum[graphcore]`   |
-| [Habana Gaudi Processor (HPU)](https://habana.ai/training/)                                                            | `python -m pip install optimum[habana]`      |
+| Accelerator                                                                                                            | Installation                                      |
+|:-----------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------|
+| [ONNX Runtime](https://onnxruntime.ai/docs/)                                                                           | `python -m pip install optimum[onnxruntime]`      |
+| Intel [Neural Compressor](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html)       | `python -m pip install optimum[neural-compressor]`|
```
Comment (JingyaHuang) [inline README.md:49]:
```suggestion
| [Intel Neural Compressor](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html)       | `python -m pip install optimum[neural-compressor]`|
```

File: docs/source/index.mdx [hunk 3]
```diff
@@ -25,7 +25,7 @@ As such, Optimum enables users to efficiently use any of these platforms with th
...
 - [Graphcore IPUs](https://github.com/huggingface/optimum-graphcore) - IPUs are a completely new kind of massively parallel processor to accelerate machine intelligence. [More information here](https://www.graphcore.ai/products/ipu).
 - [Habana Gaudi Processor (HPU)](https://github.com/huggingface/optimum-habana) - [HPUs](https://docs.habana.ai/en/latest/Gaudi_Overview/Gaudi_Architecture.html) are designed to maximize training throughput and efficiency. [More information here](https://habana.ai/training/).
-- [Intel](https://github.com/huggingface/optimum-intel) - Enabling the usage of Intel tools to accelerate end-to-end pipelines on Intel architectures. More information [here](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html).
+- [Intel](https://github.com/huggingface/optimum-intel) - Enabling the usage of Intel tools to accelerate end-to-end pipelines on Intel architectures. More information [here](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html) and [here](https://docs.openvino.ai/latest/index.html).
```
Comment (JingyaHuang) [inline docs/source/index.mdx:28]: The same here, "here and here" seems slightly ambiguous.

File: README.md (hunk 1 above)
Comment (JingyaHuang) [reply README.md:18]: I guess there might be some reasons for this(to be consistent with setup?), just feel that INC shall include Intel as its name.

File: docs/source/index.mdx [hunk 4]
```diff
@@ -53,12 +53,13 @@ python -m pip install optimum
...
-| [Graphcore IPU](https://www.graphcore.ai/products/ipu)                                                                 | `python -m pip install optimum[graphcore]`   |
-| [Habana Gaudi Processor (HPU)](https://habana.ai/training/)                                                            | `python -m pip install optimum[habana]`      |
+| Accelerator                                                                                                            | Installation                                      |
+|:-----------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------|
+| [ONNX runtime](https://onnxruntime.ai/docs/)                                                                           | `python -m pip install optimum[onnxruntime]`      |
+| Intel [Neural Compressor](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html)       | `python -m pip install optimum[neural-compressor]`|
```
Comment (JingyaHuang) [inline docs/source/index.mdx:59]: same here

**Review by JingyaHuang — APPROVED** [review]
LGTM!

**Review by regisss — APPROVED** [review]
LGTM @echarlaix !

**Pull request merged**